functions/src/days-before-campaign-ends.test.ts

```diff
@@ -11,6 +11,16 @@ test("6 days remain", t => {
   t.is(actual, 6);
 });
 
+test("6 days remain 2", t => {
+  const now = new Date(Date.UTC(2018, 8, 20,2));
```

Comment:
You can remove the fact that now is at 2AM because it is not relevant to this test and we do not want to confuse the reader as to what is relevant.